### ISSUE TITLE
Add basic build instructions to README.md

### ISSUE THREAD

**Issue by alexjlockwood** [romannurik/AndroidIconAnimator#28]
Unless you think it's super obvious. :p 

I assumed all you needed to do was:

```
npm install
```

and then

```
gulp serve
```

However, when I tried this it serves a blank website and the following output:

```
alockwood@kida:~/AndroidIconAnimator$ gulp serve
[14:03:26] Using gulpfile ~/AndroidIconAnimator/gulpfile.js
[14:03:26] Starting 'styles'...
[14:03:26] Starting 'scripts'...
[14:03:26] Starting 'icons'...
[14:03:27] Starting 'bower'...
[14:03:27] Using cwd: /usr/local/google/home/alockwood/AndroidIconAnimator
[14:03:27] Using bower dir: .tmp/lib
[14:03:28] styles all files 18.65 kB
[14:03:28] Finished 'styles' after 1.68 s
[14:03:31] icons all files 4.87 kB
[14:03:31] Finished 'icons' after 4.83 s
Error: Cannot find module '../../../_sandbox/debug.iconanim.json' from '/usr/local/google/home/alockwood/AndroidIconAnimator/app/pages/studio'
    at /usr/local/google/home/alockwood/AndroidIconAnimator/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (/usr/local/google/home/alockwood/AndroidIconAnimator/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/google/home/alockwood/AndroidIconAnimator/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/google/home/alockwood/AndroidIconAnimator/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
[14:03:34] Finished 'scripts' after 7.71 s
[14:03:37] Finished 'bower' after 11 s
[14:03:37] Starting 'serve'...
[14:03:38] Finished 'serve' after 95 ms
[BS] Access URLs:
 --------------------------------------
       Local: http://localhost:3000
    External: http://172.26.113.33:3000
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://172.26.113.33:3001
 --------------------------------------
[BS] Serving files from: .tmp
[BS] Serving files from: app
```

**Comment by romannurik** [comment]
Whoops, that error is my fault! I'll fix. Please do add that to the readme, npm install and gulp serve should really be sufficient.

**Comment by romannurik** [comment]
if you sync that local dependency should be gone

**Comment by romannurik** [comment]
Closed in #29
